source/templates.tex

```diff
@@ -4598,7 +4598,7 @@
 \item \grammarterm{parameter-declaration} in a \grammarterm{lambda-declarator}
 or \grammarterm{requirement-parameter-list},
 unless that \grammarterm{parameter-declaration} appears in a default argument, or
-\item \grammarterm{parameter-declaration} of a (non-type) \grammarterm{template-parameter}.
+\item \grammarterm{parameter-declaration} of a (constant) \grammarterm{template-parameter}.
```

Comment:
```suggestion
\item \grammarterm{parameter-declaration} of a \grammarterm{template-parameter} that declares a constant template parameter.
```

Reply:
I considered that (and also to remove the parenthetical entirely). parameter-declaration implies a constant template parameter, I'm afraid your wording might imply the existence of some alternative

Reply:
```suggestion
\item \grammarterm{parameter-declaration} of a \grammarterm{template-parameter} (which necessarily declares a constant template parameter).
```

Reply:
@cor3ntin, I think you applied the older suggestion? Jens expressed a preference for the newer one.